### PR TITLE
Consolidate contributing info

### PR DESCRIPTION
**Pull request by SKalt**
To close my take on #120. 
Technical note on the markdown changes: repeated "1."s still increment the list number, while remaining resilient to added/removed steps.  The "0." is nonstandard and meant to emphasize that finding/creating an issue should come before any code.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Contributing
+
+Thank you for your interest in `tools-python`. The project is open-source software, and bug reports, suggestions, and most especially patches are welcome.
+
+## Issues
+
+`tools-python` has a [project page on GitHub](https://github.com/spdx/tools-python/) where you can [create an issue](https://github.com/spdx/tools-python/issues/new) to report a bug, make a suggestion, or propose a substantial change or improvement that you might like to make. You may also wish to contact the SPDX working group technical team through its mailing list, [spdx-tech@lists.spdx.org](mailto:spdx-tech@lists.spdx.org).
+
+If you would like to work on a fix for any issue, please assign the issue to yourself prior to creating a patch.
```
Comment (armintaenzertng) [inline CONTRIBUTING.md:9]: I believe most people don't have the necessary rights to assign themselves to an issue

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# Contributing
+
+Thank you for your interest in `tools-python`. The project is open-source software, and bug reports, suggestions, and most especially patches are welcome.
+
+## Issues
+
+`tools-python` has a [project page on GitHub](https://github.com/spdx/tools-python/) where you can [create an issue](https://github.com/spdx/tools-python/issues/new) to report a bug, make a suggestion, or propose a substantial change or improvement that you might like to make. You may also wish to contact the SPDX working group technical team through its mailing list, [spdx-tech@lists.spdx.org](mailto:spdx-tech@lists.spdx.org).
```
Comment (armintaenzertng) [inline CONTRIBUTING.md:7]: I'd leave out the "that you might like to make". Proposals are always welcome regardless of whether the proposing person implements it itself.

File: README.md [hunk 3]
```diff
@@ -82,50 +82,22 @@ of this repo with `yourenv/bin/pip install .` or install from PyPI with
...
 # Support
 
 * Submit issues, questions or feedback at: https://github.com/spdx/tools-python/issues
-* Join the dicussion on https://lists.spdx.org/mailman/listinfo/spdx-tech and 
+* Join the dicussion on https://lists.spdx.org/mailman/listinfo/spdx-tech and
   https://spdx.org/WorkgroupTechnical
+
+# Contributing
+
+See [CONTRIBUTING.md](./CONTRIBUTING.md) for instructions on how to add contribute to the codebase.
```
Comment (armintaenzertng) [inline README.md:101]: To already welcome contributors with open arms on the front page, let's write something along the lines of "Contributions are very welcome!" here as well.

Also, there is a superfluous "add" here.

**Review by armintaenzertng — COMMENTED** [review]
Thanks, looks good :)
Just some minor remarks from my side

**Comment by SKalt** [comment]
go for it! I had forgotten about this entirely after three years.

**Pull request closed without merging**